documentation/config_file_metrics.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: metrics.conf"
+canonical: "/puppetserver/latest/config_file_metrics.html"
+---
+
+The `metrics.conf` file configures Puppet Server's [metrics services](./puppet_server_metrics.markdown) and [v2 metrics API](./metrics-api/v2/metrics_api.markdown).
+
+## Settings
+
+All settings in the file are contained in a HOCON `metrics` section.
+
+-   `server-id`: A unique identifier to be used as part of the namespace for metrics that this server produces.
+
+-   `registries`: A section that contains settings to control which metrics are reported, and how they're reported.
+    -   `puppetserver`: A section that wraps registries settings.
+        -   `metrics-allowed`: An array of metrics to report. See the [metrics documentation](./puppet_server_metrics.markdown) for details about individual metrics.
+        -   `reporters`: Can contain `jmx` and `graphite` sections with a single Boolean `enabled` setting to enable or disable each reporter type.
```

Comment:
It can contain other things (it can override settings defined in the `metrics.reporters` sections) but that's a more advanced thing for folks using multiple registries.

Don't think we want to lead people down that path (ie we only really want folks to use this section to enable/disable reporters for the `puppetserver` registry), but... I don't know why I'm writing this, you wrote exactly what we want folks to use this for.